packages/migrate/Migration.js

```diff
@@ -120,35 +115,39 @@ class Migration {
         .contracts()
         .map(abstraction => abstraction._json);
       if (
-        Db &&
         this.config.db &&
         this.config.db.enabled &&
         artifacts.length > 0
       ) {
-        const db = Db.connect(this.config.db);
-        const project = await Db.Project.initialize({
-          db,
-          project: {
-            directory: this.config.working_directory
-          }
-        });
-
-        const result = await project
-          .connect({ provider: this.config.provider })
-          .loadMigrate({
-            network: {
-              name: this.config.network
-            },
-            artifacts
+        // currently if Db fails to load, getDb returns `null`
```

Comment:
This comment would not be needed if the function name were changed to `getDbOrNull()` or something similar.